tests/test_integration.py

```diff
@@ -305,6 +305,19 @@ def test_ar():
     metrics_df = m.fit(df, freq="D")
     future = m.make_future_dataframe(df, n_historic_predictions=90)
     forecast = m.predict(df=future)
+    log.info("testing: AR only forecasting once per week instead of once per day")
+    m = NeuralProphet(
+        n_forecasts=7,
+        n_lags=7,
+        yearly_seasonality=False,
+        epochs=EPOCHS,
+        batch_size=BATCH_SIZE,
+        learning_rate=LR,
+        forecast_period=7,
```

Comment:
I think this functionality deserves its dedicated test.
I think we should also test for mismatched (overlapping and non-overlapping) `n_forecasts` - `forecast_period`  combinations, 
e.g. 
overlapping: 7-1, 7-3
matched 7-7
with gaps: 1-7, 3-7